src/com/egs/arsensargsyan/task2/ReadFromFileIntoArrayList.java

```diff
@@ -0,0 +1,110 @@
+package task2;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class ReadFromFileIntoArrayList {
+
+    private static ArrayList<User> users = new ArrayList<>();
+    private static final File FILE_PATH = new File("arsen-sargsyan-resources/fileArrayList");
+
+    public static void main(String[] args) {
+       new ReadFromFileIntoArrayList().go();
+    }
+
+    public void go() {
+        try {
```

Comment:
is better to keep these 2 methods read and write inside the same try. or let them throw exception.
because if you fail on read it's useless to continue